app/talk/comment.cjsx

```diff
@@ -76,7 +78,7 @@ module?.exports = React.createClass
     <div className="talk-comment">
       <div className="talk-comment-author">
         <PromiseRenderer pending={@avatarPending} promise={apiClient.type('users').get(id: @props.data.user_id).index(0)}>{(commentOwner) =>
-          <img src={commentOwner.avatar} alt={commentOwner.display_name}/>
+          <img src={commentOwner.avatar ? DEFAULT_AVATAR} alt={commentOwner.display_name}/>
```

Comment:
Yeah good idea- I just added in an avatar component in https://github.com/zooniverse/Panoptes-Front-End/commit/07c2429b7b5d441fe906aa785dce690f2e8e1b34. Let me know how it looks